src/core/UserIcon/UserIcon.tsx

```diff
@@ -89,35 +89,35 @@ export const UserIcon = (props: UserIconProps) => {
   const statusTitles = { ...defaultStatusTitles, ...translatedStatusTitles };
 
   const iconMap: { [key in UserIconStatus]: JSX.Element } = {
-    away: <SvgAway className='iui-user-icons-status-symbol' />,
-    offline: <SvgCloseSmall className='iui-user-icons-status-symbol' />,
-    online: <SvgCheckmark className='iui-user-icons-status-symbol' />,
+    away: <SvgAway className='iui-status-symbol' />,
+    offline: <SvgCloseSmall className='iui-status-symbol' />,
+    online: <SvgCheckmark className='iui-status-symbol' />,
     busy: <></>,
   };
 
   return (
     <span
-      className={cx(`iui-user-icons-${size}`, className)}
+      className={cx(`iui-user-icon iui-${size}`, className)}
```

Comment:
Medium size is not existing in css anymore...
But I think it makes sense to have, just to understand the sizing.
Or we could have an item to change this (to default? does not sound better...) with the next major update, but that would be only to reflect better the css. Thoughts?
Id personally be for leaving the medium in this case, to be honest.

Reply:
We can leave the prop but not apply the class?

Reply:
I guess we could, for cleanliness..